src/main/proto/wfa/measurement/internal/duchy/computations_service.proto

```diff
@@ -115,6 +116,16 @@ message UpdateComputationDetailsRequest {
   // New value of the computation_details.
   // Note that the protocol type can not be changed in the details.
   ComputationDetails details = 2;
+  // Updated requisition details.
+  message RequisitionDetailUpdate {
+    // External ID of the data provider.
+    string external_data_provider_id = 1;
+    // External ID of the requisition.
+    string external_requisition_id = 2;
+    // New value of the requisition details.
+    RequisitionDetails details = 3;
+  }
+  repeated RequisitionDetailUpdate requisition_detail_update = 3;
```

Comment:
Why is this repeated? Are Requisition detail updates batched?

Reply:
A computation has multiple requisitions, each requisition has its own detail to update. We update them together in a single rpc call.

Reply:
Makes sense, thanks